part2/stages/Functions/install-main

```diff
@@ -439,7 +439,10 @@ seal_system() {
                 do_umount ${DOM0_MOUNT}/boot/system
                 do_umount ${DOM0_MOUNT}/tmp
                 do_umount ${DOM0_MOUNT}/dev
+                do_umount ${DOM0_MOUNT}/sys/fs/selinux
```

Comment:
Is there a reason we aren't using [`do_umount_all()`](https://github.com/OpenXT/installer/blob/master/common/stages/Functions/library#L230) in this function, @dpsmith ?

Reply:
@jandryuk, not at all as I never went looking to see if such a utility function was available. I want to say I reused a pattern from the original sealing code. Either way, your right that ```do_umount_all ${DOM0_MOUNT}``` would be better.

Reply:
@jandryuk, @dpsmith, is any of you requesting changes to this PR?

Reply:
I think `do_umount_all` would be better, but it needs to be tested.  If @crogers1 wants to re-spin and test with that, then that would be fine.  Otherwise this can do in as-is and it can be revisited later.

Reply:
@jandryuk I can rebuild and test at some point today